pkg/grpc/audit/audit.pb.go

```diff
@@ -111,7 +111,7 @@ func file_audit_proto_rawDescGZIP() []byte {
 }
 
 var file_audit_proto_msgTypes = make([]protoimpl.MessageInfo, 1)
-var file_audit_proto_goTypes = []interface{}{
+var file_audit_proto_goTypes = []any{
```

Comment:
I'm not sure about modifying the generated protobuf code directly. Won't all these changes get reverted the next time someone runs `make proto`?

Reply:
Thanks, looks good to me.